MEMBERSHIP.md

```diff
@@ -0,0 +1,65 @@
+
+## Nuovi contributors
+
+Chiunque è ben accetto in community e anche a livello contributivo per l'aggiunta di informazioni al libro.
+
+I nuovi contributors dovrebbero essere accolti nella community dagli ambassador esistenti, aiutati con il workflow delle PR e guidati alla documentazione e ai canali di comunicazione pertinenti o working group di riferimento.
+
+## Contributors
+
+I Contributors sono membri costantemente attivi della community. Questi sono tenuti a rimanere attivi nella community per mantenere il livello.
+
+Questi dimostrano di aderire ai principi di questo documento, di avere familiarità con l'organizzazione del progetto, i ruoli, le politiche, le procedure, le convenzioni, ecc. Le aspettative, le responsabilità e i requisiti specifici dei ruoli sono elencati di seguito.
+
+**Definito da**: Membro dell'organizzazione GitHub di Il Libro Open Source in triage e nel team `@Il-Libro-Open-Source/Contributor` e l'ingresso nella chat Telegram della community dedicata ai Contributor.
+
+### Requisiti
+
+- Abilitazione dell'autenticazione a due fattori sul proprio account GitHub
+- Hanno apportato molteplici contributi ai progetti o alla community. Il contributo può includere, ma non è limitato a:
+  - Autorizzare o revisionare PR su GitHub
+  - Aiutare o commentare le issues su GitHub
+  - Contribuire al progetto o alle discussioni della community (e.g. riunioni, GitHub discussions)
+- Aver letto la linee guida per le community, le regole e il documento sui Contributor
+- Essere sponsorizzato da 2 (da valutare il numero) Ambassador attivi
+- Aprire un issue su questo repository utilizzando il template
+  - Assicuratevi che i vostri sponsor siano @menzionati sull'issue aperto così che possano rispondere
+  - Gli sponsor dovranno rispondere con un :+1: alla issue per accettare l'ingresso dell'utente come ambassador
+- Tutti gli ambassador saranno comunque liberi di intervenire nella discussione commentando e dando una loro personale visione che sia essa positiva o negativa
+
+### Responsabilità e privilegi
+
+- Accesso all'organizzazione GitHub in triage (ovvero con permessi di sola lettura)
+- Rispondere ai problemi e alle relative PR o issue
+- Dare assistenza a nuovi membri della community
+- Contribuire attivamente alla scrittura di capitoli interni al libro Open Source
+- Gli Ambassador possono approvare le PR aperte
+- Accesso al canale Telegram dei contributors
+- Poter richiedere di diventare Ambassador
+
+# Ambassadors
+
+Gli Ambassadors sono membri costantemente attivi della community. Questi membri sono già dei contributor molto attivi e che desiderano prendere delle responsabilità ulteriori nei riguardi del progetto.
+
+**Definito da**: Inserimento nel Team Ambassador di GitHub con i poteri di write sull'organizzazione e accesso al team GH `@Il-Libro-Open-Source/Ambassadors`.
+
+### Requisiti
+
+- Contributor da almeno 3 mesi
+- Contributor attivo di almeno 1 capitolo del Libro
```

Comment:
```suggestion
- Contributor attivo di almeno 1 capitolo del Libro
```

Questo escluderebbe chiunque possa potenzialmente passare ogni giorno della sua vita a parlare del progetto senza però scriverlo direttamente, non trovi?

Reply:
Vogliamo che possa diventare lo stesso Ambassador?

Reply:
Beh, credo possa valere la pena non escludere a prescindere l'ipotesi, non trovi?

Reply:
Comunque per essere Contributor devi contribuire attivamente al progetto, quindi qualcosa devi averlo giá scritto, se voglio diventare ambassador ci deve essere un motivo.

Potrebbe essere messo in o diventi membro di un wg o leader di almeno un capitolo del libro...